call-for-submissions.md

```diff
@@ -29,15 +29,19 @@ We do this in service of our editorial vision for this newsletter:
 
 ### What we publish
 
-We welcome a variety of content from the workers' perspective:
+We welcome a variety of contributions from the worker's perspective:
```

Comment:
The spelling of "worker's perspective" here is now inconsistent with other parts of this document, which use the spelling "workers' perspective". We've talked about this in the past, it would be good to agree on a default spelling, instead of changing it in different issues or documents.